tests/api2/pool_dataset.py

```diff
@@ -143,3 +145,72 @@ def test_13_delete_dataset():
         f'/pool/dataset/id/{dataset_url}/'
     )
     assert result.status_code == 200, result.text
+
+
+def test_14_creating_zvol():
+    global results, payload
+    payload = {
+        'name': zvol,
+        'type': 'VOLUME',
+        'volsize': 163840,
+        'volblocksize': '16K'
+    }
+    results = POST(f"/pool/dataset/", payload)
+    assert results.status_code == 200, results.text
+
+
+@pytest.mark.parametrize('key', ['name', 'type', 'volsize', 'volblocksize'])
+def test_15_verify_output_(key):
```

Comment:
Is the last underscore intentional ?

Reply:
Yes the out put look like that `api2/pool_dataset.py::test_15_verify_output_[name] PASSED`

Reply:
I can remove that no problem

Reply:
I have no objections to this, was wondering if it was a typo. Thank you for elaborating